script.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import pprint
+import config
+
+def main():
+    token = generateToken(config.client_id, config.client_secret)
+    if token:
+        response = requestVehicles(token)
+        printInfo(response)
+    return 0
```

Comment:
Why do you need to return 0 here? If we run into the invalid credentials error, this `if` block will simply be skipped and return the 0, is it the expected behavior?